XamarinCommunityToolkit/Views/CameraView/Android/CameraFragment.android.cs

```diff
@@ -330,15 +333,15 @@ void SetupImageReader()
 		{
 			DisposeImageReader();
 
-			photoReader = ImageReader.NewInstance(640, 480, ImageFormatType.Jpeg, maxImages: 1);
+			photoReader = ImageReader.NewInstance(width: photoSize.Width, height: photoSize.Height, ImageFormatType.Jpeg, maxImages: 1);
```

Comment:
is there a reason for adding property name for width and height but not for ImageFormatType. I think we can remove them at all

Reply:
I tend to think that named parameters are great for non self explainable arguments.
In this case I will keep only the `maxImages` named parameter.

Reply:
width and height param name removed